Gemfile

```diff
@@ -1,13 +1,14 @@
 source 'http://rubygems.org'
 
-ruby '2.2.4'
+ruby '2.3.1'
 
-gem 'grape'
+gem 'grape', '~> 0.16'
```

Comment:
With Gemfile.lock you don't need to specify, you could just do `bundle update grape`. But it's cool either way.